packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -130,7 +130,11 @@ class AOTSnapshotter {
     outputDir.createSync(recursive: true);
 
     final List<String> genSnapshotArgs = <String>[
+      // Shorebird uses --deterministic to improve snapshot stability and increase linking.
       '--deterministic',
+      // Shorebird dumps the class table information during snapshot compilation.
+      '--print_class_table_link_debug_info_to=${_fileSystem.path.join(outputDir.path, 'class_table.json')}',
```

Comment:
Will these overwrite each other between phases, or are the phases built to separate directories?